### PR TITLE
Feature/specific log views

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: Resources/views/process_log/_log_entry/reminder.html.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<ul class="list-group">
+    {% if message is defined %}
+    <li class="list-group-item">
+        {{ 'process_log.revision.table.log_entry.reminder.message'|trans }}: {{ message ?: '-' }}
+    </li>
+    {% endif %}
+    {% if date is defined %}
+    <li class="list-group-item">
+        {{ 'process_log.revision.table.log_entry.reminder.date'|trans }}: {{ date|date('d-m-Y h:m') }}
```
Comment (tuj) [inline Resources/views/process_log/_log_entry/reminder.html.twig]: |localizeddate('short')

File: Resources/views/process_log/_log_entry/base_economy_entry.html.twig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    </li>
+    {% endif %}
+    {% if text is defined %}
+    <li class="list-group-item">
+        {{ 'process_log.revision.table.log_entry.base_economy_entry.text'|trans }}: {{ text }}
+    </li>
+    {% endif %}
+    {% if date is defined %}
+    <li class="list-group-item">
+        {{ 'process_log.revision.table.log_entry.base_economy_entry.date'|trans }}: {{ date|date('d-m-Y h:m') }}
```
Comment (tuj) [inline Resources/views/process_log/_log_entry/base_economy_entry.html.twig]: |localizeddate('short')

File: Resources/views/process_log/_log_entry/reminder.html.twig (hunk 1 above)
Comment (lats1) [reply Resources/views/process_log/_log_entry/reminder.html.twig]: Fixed in #2924c49

File: Resources/views/process_log/_log_entry/base_economy_entry.html.twig (hunk 2 above)
Comment (lats1) [reply Resources/views/process_log/_log_entry/base_economy_entry.html.twig]: Fixed in 2924c49

**Pull request merged**